fuzy-norms/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Fuzzy Logic norms",
+  "description": "A set of T-norm and T-conorm operations for using fuzzy logic whithin BigML",
```

Comment:
i would be more explicit in that this is a way of adding new
features to a dataset...
"Computing fuzzy-logic T-norms and T-conorms as new dataset features"
or something like that